schemas/webpackOptionsSchema.json

```diff
@@ -750,7 +750,7 @@
           "type": "boolean"
         },
         "maxInitialChunkSize": {
-          "description": "Total size of all initial chunks (in kb)",
+          "description": "Total size of all initial chunks (in bytes)",
           "type": "number"
```

Comment:
Getting accurate sizes was one, and I believe in other plugins and loaders we use bytes:

* `image-loader?limit=10000`
* `AgressiveSplittingPlugin()` does bytes for min and max size.